app/views/users/show.html.erb

```diff
@@ -42,19 +37,35 @@
                     </div>
                 </div>
                 <div class="col-sm-8">
+                  <% if !@games.present? %>
+                  <div class="wrapper">
+                      <div class="one"><%= display_picture (nil) %>
+                      </div>
+                      <div class='two'>
+                        <h3 class='game-title'> No Games, Please Upload a Game </h3>
+                        <%= link_to 'New Game', new_game_path, class: "btn btn-default btn-lg pop btn" %>
+                      <% end %>
+                    </div>
                   <% if @games.present? %>
```

Comment:
Uhm. Should this just the `else` of line 40. If you make this into an `if ... else` statement, you will not need the `!` on line 40.